scripts/scripts.js

```diff
@@ -560,6 +582,24 @@ export async function lookupDocuments(pathnames) {
   return (result);
 }
 
+/**
+ * Gets details about pages that are indexed filtered by path
+ * @param {String} filter return only pages that start with this filter
+ */
+
+export async function getSearchIndex(filter) {
+  if (!window.pageIndex) {
+    const resp = await fetch(`${window.hlx.codeBasePath}/query-index.json`);
```

Comment:
Can we scope the search index for just pages within `/support/curam`? The current support search is designed for the Curam (formally SPM) product and should be filtering results from that path pattern instead of all of the pages outside of `/support/curam` or `/support/spm`.

@keith-kaplan Let me know if you agree.